strawberryfields/backends/fockbackend/ops.py

```diff
@@ -459,34 +416,16 @@ def phase(theta, trunc):
 
 
 @functools.lru_cache()
-def beamsplitter(t, r, phi, trunc, save=False, directory=None):
+def beamsplitter(t, r, phi, trunc):
```

Comment:
Curious if removing `save` and `directory` will break any of the tests :thinking:

Reply:
Looks like it doesn't!